windows/devices/uwb/include/windows/devices/uwb/UwbDeviceConnector.hxx

```diff
@@ -41,13 +37,15 @@ public:
      */
     ~UwbDeviceConnector();
 
+public:
+    // IUwbDeviceDdiConnector
     /**
      * @brief Get the name of this device.
      *
      * @return const std::string&
      */
-    const std::string&
-    DeviceName() const noexcept;
+    virtual const std::string&
+    DeviceName() const noexcept override;

```

Comment:
As mentioned earlier, let's drop this from `IUwbDeviceDdiConnector` and just expose it directly form this class.
```suggestion
    const std::string&
    DeviceName() const noexcept;
```